webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/Table/TableHooks.js

```diff
@@ -224,6 +239,11 @@ export const useBulkSelect = ({
     }
   };
 
+  const selectDefault = () => {
+    selectNone();
+    selectOptions.selectDefault();
```

Comment:
Doesn't `selectOptions.selectDefault` also does `selectNone();`?

Reply:
It does, but `selectNone` from `useBulkSelect` also clears selected results (a different set), so I thought to re-use it as well just in case something will be added later to that function.